src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerFunctionalTest.java

```diff
@@ -1228,6 +1229,16 @@ public void given_initiate_tasks_then_reconfigure_task_to_mark_tasks_for_reconfi
         common.clearAllRoleAssignments(caseworkerCredentials.getHeaders(), "WA");
     }
 
+    public void completeTaskCleanup(String taskId, String status) {
```

Comment:
Did a method that does exactly this not already exist somewhere else?  We should try and re-use that if so.